data/Publications.json

```
@@ -6832,5 +6832,17 @@
         "bibcode": "2020MNRAS.491.5925M",
         "doi": "10.1093/mnras/stz3441",
         "description": "Spectral library of age-benchmark low-mass stars and brown dwmarfs"
+    },
+    {
+        "publication": "Suar22",
+        "bibcode": "2022MNRAS.513.5701S",
+        "doi": "10.1093/mnras/stac1205",
+        "description": "Ultracool dwarfs observed with the Spitzer infrared spectrograph - II. Emergence and sedimentation of silicate clouds in L dwarfs, and analysis of the full M5-T9 field dwarf spectroscopic sample"
+    },
+    {
+        "publication": "Lipp52",
+        "bibcode": "1952ApJ...115..582L",
+        "doi": "10.1086/145584",
+        "description": "Search for Flares of DM Stars on the Sproul Astrometric Program."
     }
```

```suggestion
    }
```
Can just delete this here rather than modifying the script.